src/components/collapsable-box/collapsable-box.js

```diff
@@ -7,7 +7,8 @@ export default class extends Component {
 
   constructor(props) {
     super(props);
-    this.state = { expanded: props.expanded || false };
+    const isDesktop = window.matchMedia("(min-width: 992px)").matches;
```

Comment:
Can we make this 992px a variable of some sort? This kind of coupling between here and the actually breakpoint in the styling is absolutely going to bite us in the futuree

Reply:
yes, I'll make that change.

Reply:
done